Source/WebCore/dom/TreeScope.cpp

```diff
@@ -572,4 +577,165 @@ ExceptionOr<void> TreeScope::setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>
     return ensureAdoptedStyleSheets().setSheets(WTFMove(sheets));
 }
 
+struct SVGResouresMap {
```

Comment:
There’s a typo here in the word "resources", it’s misspelled "resoures".

Reply:
Oh, oops, fixed.